pancancer-arch-3/src/main/java/info/pancancer/arch3/coordinator/Coordinator.java

```diff
@@ -181,10 +182,11 @@ private String requestVm(String message) {
 
                 vmChannel.basicPublish("", queueName + "_vms", MessageProperties.PERSISTENT_TEXT_PLAIN,
                         message.getBytes(StandardCharsets.UTF_8));
+                vmChannel.waitForConfirms();
 
                 log.info(" + MESSAGE SENT!\n" + message + "\n");
 
-            } catch (IOException ex) {
+            } catch (IOException | InterruptedException ex) {
```

Comment:
`CoordinatorOrders` extends `Callable` so maybe it should do something else when it is interrupted by `InterruptedException`... is there any way it could gracefully clean up and shut down?